shared/charts/backup-exporter/templates/configmap.yaml

```diff
@@ -0,0 +1,21 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "backup-exporter.name" . }}
+  namespace: backup-exporter
+  labels:
+    app: {{ template "backup-exporter.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  backup.sh: |
+    gcloud config set account {{ .Values.serviceAccountName }}
+    SNAPSHOTS=$(gcloud compute snapshots list --sort-by=~creationTimestamp,STATUS --limit={{ .Values.replicaCount }} --format="value[separator=';'](name,status)" | cut -f1 -d\;)
+    DESTINATION_BUCKET={{ .Values.destinationBucket }}
+    TIMESTAMP=$(date +%s)
+    for snapshot in ${SNAPSHOTS}; do
+      gcloud -q compute images create image-disk-${snapshot} --source-snapshot ${snapshot}
```

Comment:
Should `image-disk-` prefix be configurable?

Reply:
I don't think it's worth it because it's a temporal filename, it is deleted once the copy is performed.